wgengine/magicsock/magicsock.go

```diff
@@ -2856,6 +2876,14 @@ const (
 	// goodEnoughLatency is the latency at or under which we don't
 	// try to upgrade to a better path.
 	goodEnoughLatency = 5 * time.Millisecond
+
+	// veryRarely is some long duration for a time.Ticker we don't want
+	// to actually fire.
+	veryRarely = 24 * 365 * time.Hour
```

Comment:
Oh, you mean actually really rarely. I guess this is because we can't halt the timer?

I might call this basicallyNever instead, "rarely" conjured up ideas of "once per hour" or "every few hours", not "literally once a year" :P

Reply:
PTAL. I stopped using the ticker & goroutine now and just used a time.AfterFunc.